financial-analyst-deepseek/financial-analyst-langgraph/finance_langgraph.py

```
@@ -0,0 +1,103 @@
+
+
+workflow = graph.compile()
+
+#visual representation of our graph
+#display(Image(workflow.get_graph(xray=1).draw_mermaid_png()))
+
+
+# Function to be wrapped inside MCP tool
+def run_financial_analysis(query):
+    result = workflow.invoke({"query": query})
+    result["generated_code"].content
```

Ruff (0.11.9)

Found useless attribute access. Either assign it to a variable or remove it.

(B018)